src/masonite/drivers/notification/vonage/Sms.py

```diff
@@ -0,0 +1,46 @@
+"""Vonage Sms Component"""
```

Comment:
This would be huge. I love having the idea of sending a verification code to users.